client/web/src/org/members/OrgMembersListPage.tsx

```diff
@@ -255,7 +256,7 @@ export const OrgMembersListPage: React.FunctionComponent<Props> = ({ org, authen
                     )}
                 </div>
 
-                <Container className={styles.membersList}>
+                <Container className={classNames(!showOnlyYou ? 'mb-3' : undefined, styles.membersList)}>
```

Comment:
This is shorter and more readable IMO
```suggestion
                <Container className={classNames({ 'mb-3' : !showOnlyYou}, styles.membersList)}>
```

Reply:
it's much better so i'm going to add it